### ISSUE TITLE
# tag used in heading causes overlap

### ISSUE THREAD

**Issue by thibaultmol** [insanum/obsidian_nord#60]
![image](https://github.com/insanum/obsidian_nord/assets/1010226/91f057a7-d457-4c89-9f53-c228fcb120bb)

I sometimes use tags inside of a header, and noticed that with Nord it causes the rounded box to overlap with text above and below it

**Comment by insanum** [comment]
Yeah that's annoying. It looks like the same issue occurs with the default theme. Can you verify?

**Comment by insanum** [comment]
https://forum.obsidian.md/t/tags-in-headers-overlap-other-lines-of-text-in-default-theme/48944

**Comment by thibaultmol** [comment]
Fair point yeah 
![image](https://github.com/insanum/obsidian_nord/assets/1010226/69e51b6e-4ae7-4a3e-adc1-f4a806fb4cf5)

Guess it's best to see what Obsidian itself does to fix this problem and then you can change your theme if needed. (I guess closing this issue till Obsidian fixes their theme atleast?)

**Comment by insanum** [comment]
I think it's good leave this open.

If you'd like to adjust the tag height/width, you should be able to tweak these settings in a CSS snippet:

```
    --tag-padding-x:                  .5em;
    --tag-padding-y:                  .2em;
    --tag-radius:                     .5em;
```
